src/helpers/promiseHelpers.ts

```diff
@@ -0,0 +1,3 @@
+export function wait(milliseconds: number): Promise<void> {
```

Comment:
Is `promiseHelpers.ts` an appropriate name for this file? I think just `wait.ts` would be better.

Reply:
I'm thinking it's a good file to put helper functions for various promises, if any in the future. We don't really do one file per function anywhere else in Needle so I don't think naming the file after the function follows what we do right now